instrumentation/opentelemetry-instrumentation-system-metrics/src/opentelemetry/instrumentation/system_metrics/__init__.py

```diff
@@ -79,157 +86,591 @@ def __init__(
         self.controller = PushController(
             meter=self.meter, exporter=exporter, interval=interval
         )
+        self._python_implementation = python_implementation().lower()
         if config is None:
             self._config = {
-                "system_memory": ["total", "available", "used", "free"],
-                "system_cpu": ["user", "system", "idle"],
-                "network_bytes": ["bytes_recv", "bytes_sent"],
+                "system_cpu_time": ["idle", "user", "system", "irq"],
```

Comment:
are the labels here expected to have `.` instead of `_`?

Reply:
That's right, fixing...

Reply:
Fixed